README.md

```diff
@@ -43,3 +43,7 @@ easily updated in the future as needed. Some options include:
 * [Mermaid](https://github.com/mermaid-js/mermaid#readme)
 * [Excalidraw](https://excalidraw.com/)
 ** Be sure to leave "Embed Scene" turned on when exporting the PNG.
+
+### API Specifications
+
+API definitions use [OpenAPI](https://www.openapis.org/) format in [YAML](https://yaml.org/).
```

Comment:
suggest link to the sub-dir api-definitions

```suggestion
[API definitions](./api-definitions) use [OpenAPI](https://www.openapis.org/) format in [YAML](https://yaml.org/).
```

Reply:
Thanks! @markstur can you edit your comment to have `../api-definitions` now that I put this doc under `docs/backend`? Then I can "Sign off and commit" with your name on it

Reply:
Given the conversation below about likely moving the definitions themselves to a different repo, I think I'm going to not add this suggestion for now.